.cirrus.yml

```diff
@@ -227,9 +230,9 @@ task:
         # flutter_image
         #   https://github.com/flutter/flutter/issues/100387
```

Comment:
Please add a comment here explaining why this is excluded.

Reply:
Added